hedera-evm/src/main/java/com/hedera/evm/txns/span/ExpandHandleSpanMapAccessor.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.evm.txns.span;
+
+import com.hedera.evm.usage.token.meta.FeeScheduleUpdateMeta;
+import com.hedera.evm.usage.token.meta.TokenWipeMeta;
+import com.hedera.evm.utils.accessors.TxnAccessor;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class ExpandHandleSpanMapAccessor {
```

Comment:
This accessor contains logic about transactions' context meta data. The evm library itself shouldn't be aware of this data as it only performs execution logic.